src/app/about/page.js

```diff
@@ -1,10 +1,23 @@
-import Image from "next/image";
-import Fall231 from "@/public/images/fall/fall23_1.webp";
+import Fall231 from "@/public/images/fall/fall23_1.webp"; // thisp picture was laready here when i opened file idk if this is what im supposed to put
+import { ABOUT } from "@/data/aboutPage";
+import Title from "@/components/Title";
+import Picture from "@/components/Picture";
 
 const Page = () => {
   return (
-    <div className="bg-biscuits-pink bg-blobs1 md:bg-cover bg-contain bg-repeat-y">
-      <Image src={Fall231} width={500} height={500} />
+    <div className="bg-biscuits-green-100 grid grid-cols-1 gap-y-10 p-4 md:p-10">
+      <Title text="About Us" color="red" />
+      <div className="flex justify-center">
+        <Picture x={10} y={10} image={Fall231} alt="Fall 23" />
```

Comment:
Choose a different picture other than the cupcake. Maybe a group photo would be great.

I forgot to remove the cupcake photo before pushing it to dev.

Reply:
also you can put the classname directly on the picture without creating another divider.